src/panels/lovelace/cards/hui-button-card.ts

```diff
@@ -282,6 +293,11 @@ export class HuiButtonCard extends LitElement implements LovelaceCard {
         outline: none;
       }
 
+      .state {
+        font-size: 1rem;
```

Comment:
`1rem` doesn't do anything right? This is saying: 1 x the font-size of its parent. Which it would already have?

Reply:
I was supposed to be 0.9, was playing aroudn with it :man_facepalming: